cockatrice/src/player.cpp

```diff
@@ -2602,6 +2615,90 @@ void Player::actSetPT()
     game->sendGameCommand(prepareGameCommand(commandList), playerid);
 }
 
+void Player::actChangePT() // backwards compatible with old "set" flow
+{
+    QString oldPT;
+    int playerid = id;
+
+    auto sel = scene()->selectedItems();
+    for (const auto &item : sel) {
+        auto *card = static_cast<CardItem *>(item);
+        if (!card->getPT().isEmpty()) {
+            oldPT = card->getPT();
+        }
+    }
+    bool ok;
+    dialogSemaphore = true;
+    QString pt = QInputDialog::getText(nullptr, tr("Change power/toughness"), tr("Please enter the change to PT:"),
+                                       QLineEdit::Normal, oldPT, &ok);
+    dialogSemaphore = false;
+    if (clearCardsToDelete()) {
```

Comment:
can simplify with the two if statements into one

Reply:
true, just read over that